index.js

```diff
@@ -106,13 +107,7 @@ testRunner([
 	{
 		name: 'djv',
 		setup: function (schema) {
-			var validator = djv();
-			var schemas = Object.keys(refs);
-			schemas.splice(3, 1);
-			schemas.forEach(function (uri) {
-				validator.addSchema(uri, refs[uri]);
-			});
-			return validator.addSchema('test', schema).fn;
+			return djv.addSchema('test', schema).fn;
```

Comment:
@ebdrup is it what you've meant? 
I've added a static configuration on line:35 - I add necessary schemas, excluding the draft03, as it is not supported. Also now I create a single environment as some other validators do, so in a setup I just add a schema to validate object against.